src/test/java/it/pagopa/transactions/commands/handlers/TransactionRequestAuthorizizationHandlerTest.java

```diff
@@ -155,6 +155,7 @@ void shouldSaveAuthorizationEvent() {
                 "brokerName",
                 "pspChannelCode",
                 "paymentMethodName",
+                "paymentMethodDescription",
```

Comment:
add an assertion on queue written event to check that paymentMethodDescription is effectively written into queue event

Reply:
Fixed with 👉🏻 232f60f1d7090c81fb2d1b03d23172e9a73d710d